cmake/projects/lmdbxx/hunter.cmake

```diff
@@ -0,0 +1,16 @@
+# !!! DO NOT PLACE HEADER GUARDS HERE !!!
+
+include(hunter_add_version)
+include(hunter_cacheable)
+include(hunter_download)
+include(hunter_pick_scheme)
+
+hunter_add_version(
+  PACKAGE_NAME lmdbxx
+  VERSION "0.9.14.0"
```

Comment:
Set it as default.

Reply:
Sorry messed this up after the lmdb change.